Documentation/helm-operator.md

```diff
@@ -34,26 +34,25 @@ kubectl --namespace kube-system patch deploy/tiller-deploy -p '{"spec": {"templa
 ## Installing
 
 The Ceph Operator helm chart will install the basic components necessary to create a storage platform for your Kubernetes cluster.
-After the helm chart is installed, you will need to [create a Rook cluster](quickstart.md#create-a-rook-cluster).
+After the helm chart is installed, you will need to [create a Rook cluster](ceph-quickstart.md#create-a-rook-cluster).
 
 The `helm install` command deploys rook on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation. It is recommended that the rook operator be installed into the `rook-ceph-system` namespace (you will install your clusters into separate namespaces).
 
-Rook currently publishes builds to the `alpha` and `master` channels. In the future, `beta` and `stable` will also be available.
+Rook currently publishes builds of the Ceph operator to the `beta`, `alpha`, and `master` channels. In the future `stable` will also be available.
```

Comment:
good point, i didn't think about publishing to the Beta channels now.  do we need to update our release instructions to actually perform that action?

Reply:
yes, where are those instructions?

Reply:
I believe it is a matter of running the `release/promote` pipeline in Jenkins and specifying `beta` as the channel variable in the popup dialog.  That raises an interesting question though because the makefile's promote target works across all images and charts, meaning cockroachdb and minio would also be promoted to the beta channel.  That's not really desired, I think we need to make storage provider a variable into the promote pipeline.  I'll open an issue for that.

Reply:
I've opened https://github.com/rook/rook/issues/1885 to track.  I'm considering perhaps that the Ceph CRDs should be beta versioned in this release, but we don't promote any images/charts to the Beta channels.  What do you think?

Reply:
@jbw976 the storage provider channel seems like it should match the CRD version, otherwise the versioning model seems very confusing

Reply:
Yeah you are probably right about that.  Updating the Makefile logic to only promote a specific image looks kind of complicated (or at least tricky).  If we ran the `release/promote` pipeline specifying beta as the channel with no changes to the Makefile, we'd end up with `rook/ceph:beta` but also `rook/cockroachdb:beta` and `rook/minio:beta`. Maybe we could manually delete those tags?

Reply:
Yes, seems reasonable for now to delete the tags that aren't applicable